packages/protocol/contracts/identity/FederatedAttestations.sol

```diff
@@ -83,7 +83,7 @@ contract FederatedAttestations is
   /**
    * @notice Sets the EIP712 domain separator for the Celo FederatedAttestations abstraction.
    */
-  function setEip712DomainSeparator() public {
+  function setEip712DomainSeparator() internal {
```

Comment:
not sure why this needs to be `public`, though that's how it's set in other contracts (ex. Accounts, MetaTransactionWallet)

Reply:
I think for accounts this may be because needed to be called separately through governance (since the contract was already initialized)

Reply:
going to move this down though since it's now internal